corehq/util/context_processors.py

```diff
@@ -145,8 +145,8 @@ def mobile_experience(request):
     mobile_ux_cookie_name = ''
     if (hasattr(request, 'couch_user')
         and hasattr(request, 'user_agent')
-        and (settings.SERVER_ENVIRONMENT in ['production', 'staging', 'localdev'])
-    ):
+        and (settings.SERVER_ENVIRONMENT
```

Comment:
Sorry I didn't notice on earlier reviews, but is this meant to avoid showing this on the enterprise servers? Why not use `is_saas_environment`?

Reply:
i think staging wasn't picking it up?